nix/pkgs/default.nix

```diff
@@ -154,9 +156,9 @@ in
 {
   inherit sphinx-markdown-tables sphinxemoji sphinxcontrib-haddock;
   inherit nix-pre-commit-hooks nodejs-headers;
-  inherit haskell agdaPackages cabal-install stylish-haskell hlint haskell-language-server hie-bios;
+  inherit haskell agdaPackages cabal-install stylish-haskell hlint haskell-language-server hie-bios gen-hie;
   inherit purty purty-pre-commit purs spago;
-  inherit fixPurty fixStylishHaskell updateMaterialized updateMetadataSamples updateClientDeps;
+  inherit fixPurty fixStylishHaskell updateMaterialized updateHie updateMetadataSamples updateClientDeps;
```

Comment:
this is really only a minor thing but i sometimes wonder about consistent casing:

- `gen-hie` and `update-hie`
- `genHie` and `updateHie` 

of course this also applies to other attributes all over the place.

Reply:
Yeah I agree. I made it consistent with the other things for now.

Reply:
Consistent with the inconsistency of the other things, that is!